Assets/Scripts/Input/Player.cs

```diff
@@ -169,11 +195,38 @@ private float ResolveXDistance(float xVelocity)
         RaycastHit hit;
         for (int i = -1; i <= 1; i++)
         {
-            if (Physics.Raycast(transform.position + i * new Vector3(0, height / 2f - horizRayPadding,0) , new Vector3(direction, 0, 0), out hit, width / 2f + horizRayMargin))
-                return (hit.distance -width/2) * direction;
+            if (Physics.Raycast(transform.position + i * new Vector3(0, height / 2f - horizRayPadding, 0), new Vector3(direction, 0, 0), out hit, width / 2f + horizRayMargin))
+                return (hit.distance - width / 2) * direction;
         }
         return xVelocity * Time.deltaTime;
     }
     #endregion
 
+    #region Animation
+
+    private IEnumerator AnimateJump()
+    {
+        animator.SetBool("isJumping", true);
+        yield return new WaitForSeconds(jumpToFallAnimationTime);
+        AnimateJumpToFall();
+    }
+
+    private void AnimateJumpToFall()
+    {
+        animator.SetBool("isJumping", false);
+        animator.SetBool("isFalling", true);
+        animator.SetBool("isRunning", false);
+    }
+
+    private void SetModelDirection(bool right)
```

Comment:
```suggestion
    private void SetModelRightDirection(bool right)
```
Maybe make it more clear how the function works.